internal/proxy/hook_interceptor.go

```diff
@@ -82,16 +82,33 @@ func UnaryServerHookInterceptor() grpc.UnaryServerInterceptor {
 		)
 
 		if isMock, mockResp, err = hoo.Mock(ctx, req, fullMethod); isMock {
+			log.Info("hook mock", zap.String("user", getCurrentUser(ctx)), zap.String("full method", fullMethod),
+				zap.Any("request", req), zap.Error(err))
```

Comment:
request maybe too large to print as `Info` log

Reply:
Generally, this will not be called. It will only be called when hook.so mocks a request. This info log is mainly for troubleshooting when the user encounters something that does not meet expectations, and to explain some phenomena to the user.